components/contribution-flow/StepDetailsCrypto.js

```diff
@@ -71,30 +51,27 @@ const validateCryptoCurrencyAmount = (touched, amount, minimumAmount, cryptoCurr
 
 const StepDetailsCrypto = ({ onChange, data, collective }) => {
   const intl = useIntl();
+
   const [selectedCryptoCurrency, setSelectedCryptoCurrency] = useState(data.currency);
   const [amount, setAmount] = useState(data.amount);
   const [convertedAmount, setConvertedAmount] = useState(null);
-  const [cryptoExchangeRate, setCryptoExchangeRate] = useState(null);
   const [touched, setTouched] = useState(false);
   const collectiveCurrency = collective.currency;
   const dispatchChange = (field, value) => {
     onChange({ stepDetails: { ...data, [field]: value }, stepSummary: null });
   };
 
-  const storeCryptoExchangeRate = async (cryptoCurrency, collectiveCurrency) => {
-    const exchangeRate = await getCryptoExchangeRate(cryptoCurrency, collectiveCurrency);
-    setCryptoExchangeRate(exchangeRate);
-  };
+  const { data: fxData } = useQuery(cryptoExchangeRateQuery, {
```

Comment:
`fxData` is not a great name

Reply:
I've changed this to `cryptoExchangeRateData` instead.